### ISSUE TITLE
Feature: Play All Button

### ISSUE THREAD

**Issue by daviddarnes** [EliotSlevin/NostalgiaMachine#1]
Might be a tall order, you'll probably have to make a YouTube playlist.
Be awesome addition though.

**Comment by EliotSlevin** [comment]
Yeah I was totally thinking about that. Just simple auto play could solve this, so when a video ends the next one loads up and plays, but not quite sure how/where to start on that. 

Making a youtube playlist is an option, a script could go through the pages and add each videoid into a youtube playlist via the youtube api, it would only have to be done once. Only problem is when videos get updated.

**Comment by adamransom** [comment]
You can use the [YouTube iFrame API](https://developers.google.com/youtube/iframe_api_reference#Events) to check when a video has finished playing and then start another one :) Embedly also has a way to subscribe to player events using its [player.js](https://github.com/embedly/player.js).
